app/res/layout-land/mainnew.xml

```diff
@@ -78,15 +79,19 @@
                             android:text="Saved" />
                     </LinearLayout>
                 </LinearLayout>
-
+                <!-- empty view with weight 1 to ensure str_version is bottom-aligned -->
+                <View

```

Comment:
Odd workaround - could you find an SO post or anything on why this is necessary?